packages/internal-paths/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "@redwoodjs/internal-paths",
+  "version": "3.2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/redwoodjs/redwood.git",
+    "directory": "packages/internal-paths"
+  },
+  "license": "MIT",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "yarn build:js && yarn build:types",
+    "build:clean-dist": "rimraf 'dist/**/*/__tests__'",
+    "build:js": "babel src -d dist --extensions \".js,.ts,.tsx\" --copy-files --no-copy-ignored && yarn build:clean-dist",
+    "build:types": "tsc --build --verbose",
+    "build:watch": "nodemon --watch src --ext \"js,ts,tsx\" --ignore dist --exec \"yarn build\"",
+    "fix:permissions": "chmod +x dist/generate/generate.js dist/generate/watch.js",
+    "prepublishOnly": "NODE_ENV=production yarn build",
+    "test": "jest src",
+    "test:watch": "yarn test --watch"
+  },
+  "dependencies": {
```

Comment:
@jtoar can we confirm via Yarn whether or not all these packages are required in this new internal-paths package? Without having looked at the code yet, I'm a bit surprised by a few of these.

Reply:
@thedavidprice we only need a fraction of these; see [link tbd]